scripts/adaptor.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import pika
+import random
+import json
+
+connection = pika.BlockingConnection(
+    pika.ConnectionParameters(host='localhost'))
+
+channel = connection.channel()
+
+channel.queue_declare(queue='rpc_queue')
+channel.exchange_declare(exchange='device_commands', exchange_type='direct')
+
+deviceIDs = {'device_0':0, 'device_1':1, 'device_2':2, 'device_3':3, 'device_4':4}
+queue_names = []
+for deviceTitle in deviceIDs.keys():
+    result = channel.queue_declare(queue='')
+    queue_name = result.method.queue
+    queue_names.append(queue_name)
+    channel.queue_bind(
+        exchange='device_commands', queue=queue_name, routing_key=deviceTitle)
+
+def generate_status():
+    return random.choice([True, False])
+
+def generate_command_status():
+    return random.choice(['Executing', 'Finished', 'Queued'])
+
+def getDevicesStatus():
+    # {'_id': 0, 'title': 'device_0', 'isConnected': True}
+    deviceID = {'device_0':0, 'device_1':1, 'device_2':2, 'device_3':3, 'device_4':4}
+    devicesStatusList = []
+    
+    for deviceTitle, deviceId in deviceID.items():
+        status = generate_status()
+        devicesStatusList.append({'_id': deviceId, 'title': deviceTitle, 'isConnected': status})
+    return devicesStatusList
+
+def on_request(ch, method, props, body):
+    print(f" [.] incomming message: {body}")
+    # response = fib(n)
+    status = json.dumps(getDevicesStatus())
+
+    ch.basic_publish(exchange='',
+                     routing_key=props.reply_to,
+                     properties=pika.BasicProperties(correlation_id = \
+                                                         props.correlation_id),
+                     body=status)
+    ch.basic_ack(delivery_tag=method.delivery_tag)
+
+def on_command_request(ch, method, props, body):
+    json_body = json.loads(body)
+    print(f" [.] incomming command: {json_body}")
+    # {'command_id': str(uuid.uuid4()), 'command': 'test_command', 'device_id': 'id_1'}
+    device_id = json_body['device_id']
+    command_id = json_body['command_id']
+    device_status = generate_command_status()
```

Comment:
Can you put a swtich on the device_commands?

Reply:
I'm sorry I don't understand, what do you mean a switch?

Reply:
Added some dummy if else for different dummy commands